docs/getting-started/automatic-import.asciidoc

```diff
@@ -57,6 +65,12 @@ image::images/auto-import-review-integration-page.png[The Automatic Import Revie
 +
 12. (Optional) After reviewing the proposed pipeline, you can fine-tune it by clicking **Edit pipeline**. Refer to the <<siem-field-reference,{elastic-sec} ECS reference>> to learn more about formatting field mappings. When you're satisfied with your changes, click **Save**. 
 +
+[NOTE]
+.How to edit a CEL program
+====
+If your new integration collects CEL data, you must install it before you can edit its CEL program. After installation, you can update the CEL program and API authentication information from the new integration's integration policy.
```

Comment:
CEL is the method by which we collect data from an API, not a type of data we collect. 

And they can edit things before installing or after. The editing occurs on the integration policy add/edit page. 

```suggestion
If your new integration collects data from an API, you can update the CEL input configuration  (program and API authentication information) from the new integration's integration policy.
```